pkg/logs/auditor/auditor.go

```diff
@@ -104,7 +104,7 @@ func (a *RegistryAuditor) Stop() {
 func (a *RegistryAuditor) createChannels() {
 	a.chansMutex.Lock()
 	defer a.chansMutex.Unlock()
-	a.inputChan = make(chan *message.Payload, config.ChanSize)
+	a.inputChan = make(chan *message.Payload, pkgconfigsetup.Datadog().GetInt("logs_config.message_channel_size"))
```

Comment:
Ideally we should provide the config component as part of `New` function, so we do not use the `pkgconfigsetup.Datadog()` global 😄 

Maybe something we could do on a separate PR